src/system-script.js

```diff
@@ -40,7 +24,7 @@ function getScript(address) {
 }
 
 export function fetch(load) {
-	outerSystem = this;
+	outerSystem = window.__systemAmdScript.SystemJS = this;
```

Comment:
Make sure that both files are sharing the same version of systemjs. This is important for the tests.